index.html

```diff
@@ -68,18 +68,34 @@ <h2 id="introduction">Introduction</h2>
-<p><dfn>Base direction</dfn>  determines the general arrangement and progression of content when bidirectional text is displayed. The <dfn data-lt="unicode bidirectional algorithm|uba">Unicode Bidirectional Algorithm</dfn> or <q>UBA</q> [[UAX9]] is primarily focused on arranging adjacent characters, based on character properties. Base direction works at a higher level, and dictates (a) the visual order and direction in which runs of strongly-typed LTR and RTL character are displayed, and (b) where there are weakly-typed characters such as punctuation, the placement of those items relative to the other content.</p>
+<p class="definition"><dfn>LTR</dfn> stands for "left-to-right" and refers to the inline base direction of left-to-right [[UAX9]]. This is the base text direction used by languages whose starting character progression begins on the left side of the page in horizontal text. It's used for scripts such as Latin, Cyrillic, Devanagari, and many others.</p>
   
-  <p class="note">If you are unfamiliar with bidirectional or right-to-left text, there is a basic introduction <a href="https://www.w3.org/International/articles/inline-bidi-markup/uba-basics">here</a>. This will give you a basic grasp of how the <a>Unicode Bidirectional Algorithm</a> works and the interplay between it and the base direction, which will stand you in good stead for reading this document. Additional materials can be found in the Internationalization Working Group's <a href="https://www.w3.org/International/techniques/developing-specs.en?open=text_direction#text_direction">Techniques Index</a>.</p>
+<p class="definition"><dfn>RTL</dfn> stands for "right-to-left" and refers to the inline base direction of right-to-left [[UAX9]]. This is the base text direction used by languages whose starting character progression begins on the right side of the page in horizontal text. It's used for a variety of scripts which include Arabic, Hebrew, N'Ko, Adlam, Thaana, and Syriac among others.</p>
+
+<p class="definition"><dfn data-lt="unicode bidirectional algorithm|uba|bidi algorithm">Unicode Bidirectional Algorithm</dfn> (<q>UBA</q>) or <em>Bidi algorithm</em>. This is the name for the rules described in <a href="http://www.unicode.org/reports/tr9/"><cite>Unicode Standard Annex #9: <q>Unicode Bidirectional Algorithm</q></cite></a> [[UAX9]]. Those rules describe how inline bidirectional text should be rendered. The effects of the bidi algorithm depend on the <a>base direction</a> and the directional properties of the characters to which it is applied.</p>
+
+<p class="definition"><dfn>Base direction</dfn> is the initial direction applied to a paragraph and determines the general arrangement and progression of content in the <a>bidi algorithm</a> when bidirectional text is displayed.</p>
```

Comment:
base direction is the initial direction applied to both paragraphs and inline runs.  In this document, however, we are only concerned with the base direction of a string as a whole.  That's probably similar to paragraph level when the string is stored in a manifest, but it may well become the direction of an inline run when included in the rendered text.   Perhaps we should say something like:

"Base direction is the initial direction applied to a paragraph or a defined run of inline text, and determines .... In this document we are concerned with identifying the base direction of a whole string, and we do not talk about how to define base direction for runs of text within a string."

Reply:
I don't understand "base direction" as being applied to "inline runs" save where we are speaking of isolated runs of text. Otherwise isn't the direction of a run of text its embedding level over the base direction? 

That would make your suggestion more like "... In this document we are concerned with identifying the base direction of a whole string and to apply that base direction with displaying strings in a given context, and we do not talk about how to determine the direction of runs of text within a string."

Reply:
Embedding levels change the base direction for an inline run of text.  So if one is defining what base direction is, one needs to take that into account.  Of course, the more important reason for having a definition of base direction in this document is to say what's out of scope for the document - ie. embedded base direction changes within a string.

Reply:
Hm. I never call what embedding levels are changing the "base direction". I just call it the direction of that run. The base direction (in my mind) is the *floor* (whether one is starting with `L` or `R` as the direction or `0` or `1` as the level I suppose).

It is super unhelpful that UTR#9 contains no definitions for this stuff (I looked) and I suspect we should contribute definitions to the *Unicode* glossary as well. Have a look at my edits just now and see what you think?